stripper/ze_zombierig_p3.cfg

```diff
@@ -134,4 +135,41 @@ modify:
 filter:
 {
 	"classname" "point_viewcontrol"
-}
\ No newline at end of file
+}
+
+;Removing map internal weapon-cleaner since (most) servers have their own weapon-cleaning plugin nowdays.
+;Map internal weapon-cleaner does not take guns with hammerids into account thus deleting items upon dropping making them impossible to recover.
+;Remove these lines below if your server does not have its own weapon-cleaner plugin to avoid lag.
+
+filter:
+{
+	"targetname" "WeaponManager"
+	"hammerid" "380948"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "117952"
+	}
+	delete:
+	{
+		"OnStartTouch" "WeaponManagerFireUser151"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "normal_once"
+		"hammerid" "44698"
+	}
+	delete:
+	{
+		"OnStartTouch" "WeaponManagerFireUser1401"
```

Comment:
This output doesn't "exist", the real one has 2 escape characters after FireUser1 because that's standard syntax.